src/main/java/club/sk1er/patcher/mixins/bugfixes/optifine/RegionRenderCacheMixin_FixConnectedTextures.java

```diff
@@ -0,0 +1,36 @@
+//#if MC==10809
+package club.sk1er.patcher.mixins.bugfixes.optifine;
+
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.client.renderer.RegionRenderCache;
+import net.minecraft.util.BlockPos;
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+@SuppressWarnings("UnusedMixin") //added in PatcherMixinPlugin
```

Comment:
Unnecessary, our MixinPlugin stops anything in a `.optifine` package from being applied if OptiFine isn't found.